d3ledger/back-office
# D3-776 | Update to RC2

# Description
The task is about to update front-end for new version of iroha. 🤕

#### Works done
- [x] Updated queries 😄
- [x] Updated commands 😃
- [x] Updated genesis block 🙂
- [x] Updated unit test due to new transaction response 😐
- [x] Updated setup script ☹️
- [x] Error with exchange 😡

# How to test
1. `yarn` to install new dependencies
2. `yarn serve` to run application
3. `yarn init-fake-tokens` to fill iroha with fake erc20 tokens
4. Test all features!

Mood 🤯




Oh god... It should work!
